product_configurator/models/product_attribute.py

```diff
@@ -110,27 +112,27 @@ def validate_custom_val(self, val):
                 raise ValidationError(
                     _(
                         "Selected custom value '%(name)s' must be "
-                        "between %(min_val)s and %(max_val)s"
-                    )
-                    % (
-                        {
+                        "between %(min_val)s and %(max_val)s",
+                        **{
                             "name": self.name,
                             "min_val": self.min_val,
                             "max_val": self.max_val,
-                        }
+                        },
                     )
                 )
             elif minv and val < minv:
                 raise ValidationError(
-                    _("Selected custom value '%(name)s' must be at least %(min_val)s")
-                    % ({"name": self.name, "min_val": self.min_val})
+                    _(
+                        "Selected custom value '%(name)s' must be at least %(min_val)s",
+                        **{"name": self.name, "min_val": self.min_val},
+                    )
```

Comment:
I confess this style of passing translation string arguments is unusual...

Reply:
It is just the new way of doing it :) have a look at https://github.com/odoo/odoo/commit/3c3b88f6768e31417263b3828a874ca204e29514